saleor/app/installation_utils.py

```diff
@@ -96,7 +98,8 @@ def fetch_icon_image(
                 content.write(chunk)
                 if content.tell() > max_file_size:
                     raise ValidationError(size_error_msg, code=code)
-                if (time.monotonic() - fetch_start) > timeout:
+                timeout_in_secs = sum(timeout)
```

Comment:
Here it would make more sense to use `timeout[1]` since it's the read timeout and we are reading